packages/flutter/lib/src/widgets/fade_in_image.dart

```diff
@@ -20,30 +21,30 @@ import 'ticker_provider.dart';
 ///
 /// Use this class to display long-loading images, such as [new NetworkImage],
 /// so that the image appears on screen with a graceful animation rather than
-/// abruptly pops onto the screen.
+/// abruptly popping onto the screen.
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=pK738Pg9cxc}
 ///
 /// If the [image] emits an [ImageInfo] synchronously, such as when the image
-/// has been loaded and cached, the [image] is displayed immediately and the
+/// has been loaded and cached, the [image] is displayed immediately, and the
 /// [placeholder] is never displayed.
 ///
 /// [fadeOutDuration] and [fadeOutCurve] control the fade-out animation of the
```

Comment:
nit (while you're in here, but feel free to ignore since it wasn't introduced by this PR): Documentation should use proper grammar and and sentences shouldn't start with a lower case character, see style guide.

(here and below)

Reply:
Done